storage/opsmgr.go

```diff
@@ -154,34 +166,130 @@ func (v *OmAccessor) Get(path string) (map[string]string, error) {
 //List attempts to list the paths in the ops manager that could have certs
 func (v *OmAccessor) List() (PathList, error) {
 	var finalPaths []string
-	deployments, err := v.getDeployments()
+	path := fmt.Sprintf("/api/v0/deployed/certificates")
```

Comment:
drop the Sprintf, make this a const string

Reply:
Yupp good catch